web/src/scripts/containers/TabbedEditor.jsx

```diff
@@ -171,6 +176,43 @@ class TabbedEditor extends Component {
       this.setState({files})
       this.openMenu(MENU_FILE_SEARCH)
     },
+    focusNextGroup: (e) => {
+      const { dispatch } = this.props
+      dispatch(tabActions.focusAdjacentGroup(CONTENT_PANES.CENTER, (i) => i+1))
+    },
+    focusPrevGroup: (e) => {
+      const { dispatch } = this.props
+      dispatch(tabActions.focusAdjacentGroup(CONTENT_PANES.CENTER, (i) => i-1))
+    },
+    nextTab: (e) => {
+      const { dispatch } = this.props
+      dispatch(tabActions.focusAdjacentTab(CONTENT_PANES.CENTER, (i) => i+1))
+    },
+    prevTab: (e) => {
+      const { dispatch } = this.props
+      dispatch(tabActions.focusAdjacentTab(CONTENT_PANES.CENTER, (i) => i-1))
+    },
+    openInNewTab: (e) => {
+      const {decoDoc, dispatch} = this.props
+      if (!decoDoc) return
+      dispatch(tabActions.splitRight(CONTENT_PANES.CENTER, URIUtils.filePathToURI(decoDoc.id)))
```

Comment:
we probably want this for non-file tabs too, so let's use the `tabId` directly, instead of building from a `decoDoc`